index.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Kinto React Boilerplate</title>
-   <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+  <title>I really hate money</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
If possible, I would let it like it is rather than merging the change: from the tests I've done, it works better with the old rule for mobile.